package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "wrangler-action",
-	"version": "3.3.2",
+	"version": "3.4.0",
```

```suggestion
	"version": "3.3.2",
```

I should not have edited the version directly here according to https://github.com/cloudflare/wrangler-action/pull/213#discussion_r1424532205